src/main/java/com/odysseusinc/athena/service/impl/AsyncVocabularyService.java

```diff
@@ -102,8 +106,12 @@ public void saveContent(DownloadBundle bundle, AthenaUser user) {
 
             LOGGER.info("Bundle is saved in zip: {}", bundle);
             updateStatus(bundle, DownloadBundleStatus.READY);
+
+            final Map<String, String> includedVocabularies = bundle.getVocabularies().stream().map(DownloadItem::getVocabularyConversion)
+                    .filter(vocab -> !vocab.getOmopReqValue()).collect(Collectors.toMap(VocabularyConversion::getIdV5, VocabularyConversion::getName));
```

Comment:
can you formatted this expression? It is tricky to read. Please write every operation on a new line.

Reply:
done